### ISSUE TITLE
report-needless-disables ignores eslint-disable

### ISSUE THREAD

**Issue by saiichihashimoto** [stylelint/stylelint#4190]
> Clearly describe the bug

When using `--report-needless-disables`, it ignores all of my `eslint-disable`s.

> Which rule, if any, is the bug related to?

all

> What CSS is needed to reproduce the bug?

```css
/* stylelint-disable selector-max-type */
/* stylelint-disable selector-max-universal */

*,
::after,
::before {
	box-sizing: border-box;
	padding: 0;
	margin: 0;
}

a {
	color: inherit;
	text-decoration: inherit;
}

```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "selector-max-type": 0,
    "selector-max-universal": 0
  }
}
```

> Which version of stylelint are you using?

`d3a6cb47fc2d5c59e61f6fac618c6121b316b97c`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

`stylelint --fix --color --allow-empty-input --report-needless-disables`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

The problem applies to both

> What did you expect to happen?

For it to disable the rules

> What actually happened (e.g. what warnings or errors did you get)?

They were not disabled

**Comment by evilebottnawi** [comment]
What is `d3a6cb47fc2d5c59e61f6fac618c6121b316b97c` version?

How `report-needless-disables` related to `eslint` comments?

**Comment by hudochenkov** [comment]
It's a known issue. `--fix` ignores `stylelint-disable` comments https://github.com/stylelint/stylelint/issues/2643.

**Comment by saiichihashimoto** [comment]
I meant stylelint-disable, but good to know.
